datadog_cluster_agent/metadata.csv

```diff
@@ -0,0 +1,19 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
+datadog.cluster_agent.admission_webhooks.certificate_expiry,gauge,,hour,,Time left before the certificate expires in hours.,0,datadog_cluster_agent,admission webhooks certificate expiry
+datadog.cluster_agent.admission_webhooks.mutation_attempts,gauge,,,,"Number of pod mutation attempts by mutation type (agent config, standard tags).",0,datadog_cluster_agent,admission webhooks mutation attempts
+datadog.cluster_agent.admission_webhooks.reconcile_success,gauge,,,,Number of reconcile success per controller.,0,datadog_cluster_agent,admission webhooks reconcile success
+datadog.cluster_agent.admission_webhooks.webhooks_received,gauge,,request,,Number of mutation webhook requests received.,0,datadog_cluster_agent,admission webhooks received
+datadog.cluster_agent.cluster_checks_busyness,gauge,,,,Busyness of a node per the number of metrics submitted and average duration of all checks run,-1,datadog_cluster_agent,cluster check node busyness
```

Comment:
not sure about the meaning of `orientation=-1` here, maybe someone from agent integrations knows?

Reply:
Hey @ahmed-mez, I just wanted to quickly clarify what `orientation` refers to. It refers to whether the metric is positive or negative, meaning "is it a good thing if the value of this metric increases?" So something like `error_count` would have `orientation=-1` since we don't want a high error count, while something like `successful_call_rate` would have `orientation=1` since having a higher successful call rate would be a good thing.